src/controllers/tutor.js

```diff
@@ -1,8 +1,49 @@
+const { studentModel } = require('../models/student');
+
 module.exports = {
-  getAllTutors: async () => {},
-  getTutorDetails: async () => {},
-  filterTutorsByTags: async () => {},
-  searchForTutor: async () => {},
+  getAllTutors: async (req, res) => {
+    try {
+      const tutors = await studentModel.find({ isTutor: true });
+      res.json(tutors);
+    } catch (e) {
+      res.status(400).send(e);
```

Comment:
I think sending the error message within a json object will be better for parsing the response, e.g. `.json({ message: e.message })`